components/brave_wallet_ui/common/hooks/assets-management.ts

```diff
@@ -22,7 +22,6 @@ export default function useAssetManagement (
   setUserAssetVisible: SimpleActionCreator<SetUserAssetVisiblePayloadType>,
```

Comment:
@Douglashdaniel **General remark:** I believe all occurrences of `findVisibleTokenInfo` or wherever we match tokens by contract address, we need to add a condition for `chainId` as well. Can you confirm and do a quick audit if applicable?